pyro/infer/svi.py

```diff
@@ -103,7 +103,11 @@ def evaluate_loss(self, *args, **kwargs):
         Evaluate the loss function. Any args or kwargs are passed to the model and guide.
         """
         with torch.no_grad():
-            return torch_item(self.loss(self.model, self.guide, *args, **kwargs))
+            loss = self.loss(self.model, self.guide, *args, **kwargs)
+            if isinstance(loss, tuple):
+                return torch_item(loss[0]), torch_item(loss[1])
```

Comment:
ditto: can you support longer and namedtuples here:
```py
return type(loss)(map(torch_item, loss))
```

Reply:
Ah, missed that one. Done in 60e277a3.